src/app/+dashboard/data/challengesData.ts

```diff
@@ -0,0 +1,39 @@
+export const availableChallenges = [
+  {
+    chName: '10 cats',
```

Comment:
it is not obvious property name, please change it

Reply:
fixed